src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/model/ManagedIndexMetaData.kt

```diff
@@ -147,31 +65,47 @@ data class ManagedIndexMetaData(
             STEP_START_TIME to stepStartTime?.toString(),
             STEP_COMPLETED to stepCompleted?.toString(),
             FAILED to failed?.toString(),
+            POLICY_COMPLETED to policyCompleted?.toString(),
+            ROLLED_OVER to rolledOver?.toString(),
             INFO to info?.let { Strings.toString(XContentFactory.jsonBuilder().map(it)) },
             CONSUMED_RETRIES to consumedRetries?.toString()
         )
     }
 
     override fun toXContent(builder: XContentBuilder, params: ToXContent.Params): XContentBuilder {
+        // Order matters here as we are only trying to show what is needed for the customer
```

Comment:
I am not sure about what you mean with `Order matters here`. The order here is just how the json will look when creating the respond. I think you might have misunderstood with order of `writeTo` and `fromStreamInput`. That is where the order really matter. If we have incorrect write and read we will be assign incorrect value to a variable.

Reply:
The order I'm talking about is the order in which we check the values. If the policyCompleted is true then we do not want to show any of the other fields to the user and can return early.